tests/framework/cli/test_cli.py

```diff
@@ -719,6 +719,51 @@ def test_fail_reformat_load_versions(self, fake_project_cli, fake_metadata):
         )
         assert expected_output in result.output
 
+    @mark.parametrize(
```

Comment:
If `split_node_names` is an importable function, does it make more sense to not have a CLI test, but unit test the function instead?

Reply:
This would be nice, but we don't really have separate tests for any of the other "util" cli functions. All of that functionality is tested through CLI tests. So I think for consistency it makes sense to test this functionality in a CLI test as well.

Reply:
I think this is a really good question. As @merelcht says, I think the current set up fits our strategy for testing `utils`, and I actually personally think the current strategy is the right way to go:

1. It tests two parts of the code base at once, which reduces development and testing burden.
2. It can can pick up broken integrations.

Since this is a (maybe rare) instance where writing integration tests is easier / faster than writing unit tests, I think we should stick with it.